newIDE/app/src/ProjectCreation/EmptyAndStartingPointProjects.module.css

```diff
@@ -0,0 +1,25 @@
+.container {
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+}
+
+.emptyProject {
+  margin: 4px;
+  border: 1px solid var(--theme-text-disabled-color);
+  border-radius: 8px;
+  height: 100%;
+  /* In case the images cannot load, while being offline for instance */
+  min-height: 70px;
+  padding: 4px;
+  display: flex;
+}
+
+.emptyProject:hover {
+    background-color: var(--theme-hover-background-color);
+}
+
+.emptyProject:focus {
+    background-color: var(--theme-hover-background-color)
```

Comment:
Feels weird to have a hover color that is not linked to a status (warning, success) or a component (surface dark or light)

Reply:
Could you add a transition on the background-color also?

Reply:
Yeah it feels like we can't use the existing palette color for hover (from MUI I suppose?) inside the css file, so I created a prop for it, so we can use it everywhere. 
It seems like this will be a color we can use across all components